edalize/incisive.py

```diff
@@ -0,0 +1,64 @@
+import os
+import logging
+
+from edalize.edatool import Edatool
+
+logger = logging.getLogger(__name__)
+
+
+class Incisive(Edatool):
+
+    _description = """ Cadence Incisive Verilog Backend
+
+Example snippet of a CAPI2 description file for Incisive:
+
+.. code:: yaml
+
+   incisive:
+     incisive_options:
+       # Compile-time options passed to the incisive command
+       - -some_buildarg
+     run_options:
+       # Run-time options passed to the simulation itself
+       - +some+plusarg
+"""
+
+    tool_options = {
```

Comment:
tool_options and _description should be replaced with a get_doc function. I have missed that in my review of the spyglass, vcs and ascentlint backends but you could see any of the other backends. It should be pretty self-explanatory.